public/src/components/epicTests/epicTestVariantEditor.tsx

```diff
@@ -117,20 +119,26 @@ class EpicTestVariantEditor extends React.Component<Props> {
           <Typography>Fill out each field before publishing your test</Typography>
 
           <EditableTextField
-            required
```

Comment:
Should we move this field now that it's not required?

Reply:
Since there's only one mandatory field (paragraphs) and it has `Must not be empty` underneath it, I think we can remove the 'Required' + 'Optional' headings. What do you think?

Reply:
Yes - we can explain to UX team why that makes more sense now.

Reply:
👍 I've updated the screenshots

Reply:
Great!!